crates/re_sdk_comms/src/lib.rs

```diff
@@ -29,7 +29,6 @@ pub fn default_server_addr() -> std::net::SocketAddr {
 }
 
 /// The default amount of time to wait for the TCP connection to resume during a flush
-#[allow(clippy::unnecessary_wraps)]
-pub fn default_flush_timeout() -> Option<std::time::Duration> {
-    Some(std::time::Duration::from_secs(2))
+pub fn default_flush_timeout() -> std::time::Duration {
+    std::time::Duration::from_secs(2)
 }
```

Comment:
This makes the api actually harder to use and it seems the only unwrap this removes is on a pyo3 macro. That's the wrong tradeoff for me, we shouldn't (break api and) make it harder to use just because of this. Instead let's make a local exception

Reply:
Argh...I'm really sorry, didn't consider that it's part of the sdk 😬 just found it very weird and confusing that an `Option` is returned here when it always returns a `Some`.